src/renderer/components/Feed/Feeds/CameraFeed.tsx

```diff
@@ -90,23 +90,35 @@ const Webcam: FC<{ deviceid: string; flipped: boolean; rotated: boolean }> = ({
 const View: FC<Props> = ({ feed }) => {
   const source = useSelector(selectVideoUrl(feed.camera))
   const imageRef = useRef<HTMLImageElement | null>(null)
+  const [frame, setFrame] = useState('')
+
+  useEffect(() => {
+    const refCopy = imageRef.current
+    // Directly using the source redux state is causing issues so a react state is used instead
+    setFrame(source)
```

Comment:
Can you explain this a bit? I'm not sure I understand the issue.

Reply:
Interesting, that's a bit weird indeed, but if it works now that's all that matters.